File: SproInstall/SocketPro/tutorial/VBNet/MySecure/MySecureServer/MyBlowFish_i.vb
Public Class MyBlowFishConst
	'defines for service CMySecure
    Public Const sidCMySecure As Integer = (CInt(USOCKETLib.tagOtherDefine.odUserServiceIDMin) + 101)

    Public Const idOpenCMySecure As Short = (CShort(USOCKETLib.tagOtherDefine.odUserRequestIDMin) + 0)
	Public Const idBeginTransCMySecure As Short = (idOpenCMySecure + 1)
	Public Const idExecuteNoQueryCMySecure As Short = (idBeginTransCMySecure + 1)
	Public Const idCommitCMySecure As Short = (idExecuteNoQueryCMySecure + 1)
End Class

File: SproInstall/SocketPro/tutorial/CSharp/MySecure/MySecureServer/MyBlowFish_i.cs
public class MyBlowFishConst
{
	//defines for service CMySecure
	public const int sidCMySecure = ((int)USOCKETLib.tagOtherDefine.odUserServiceIDMin + 101);

	public const short idOpenCMySecure = ((short)USOCKETLib.tagOtherDefine.odUserRequestIDMin + 0);
	public const short idBeginTransCMySecure = (idOpenCMySecure + 1);
	public const short idExecuteNoQueryCMySecure = (idBeginTransCMySecure + 1);
	public const short idCommitCMySecure = (idExecuteNoQueryCMySecure + 1);
}